src/draw/control/Draw.RemoveLastPoint.js

```diff
@@ -0,0 +1,15 @@
+L.Draw.RemoveLastPoint = L.ToolbarAction.extend({
+	options: {
+		toolbarIcon: { html: 'Delete last point' }
```

Comment:
These string should be using the `L.drawLocal` object to handle people wanting to easily add other language support.